contentcuration/contentcuration/views/base.py

```diff
@@ -174,13 +174,20 @@ def channel_list(request):
                                                  })
 
 
+def _apply_channel_filters(channels, params):
```

Comment:
Is the `published` param the only one that's used? Do you foresee this channel filter getting extended more in the future?